index.html

```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
```

Comment:
I don't even know, I'm trying to add a suggestion and I couldn't find a way to do it outside of this.

Reply:
I don't see where to do the last part of this tutorial and it's really annoying.